scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -578,43 +595,34 @@ private void initSubmitAtPart() {
         startNowRadioButton.setValue(true);
         startAtRadioButton.setValue(false);
 
-        startAccordingPlanningRadioButton.addClickHandler(new com.google.gwt.event.dom.client.ClickHandler() {
-            @Override
-            public void onClick(com.google.gwt.event.dom.client.ClickEvent event) {
-                if (isExecCalendarValueNull) {
-                    displayErrorMessage("EXECUTION_CALENDAR value is empty.");
-                }
-
-                startAtLayout.removeMember(dateChooser);
+        startAccordingPlanningRadioButton.addClickHandler(event -> {
+            if (isExecCalendarValueNull) {
+                displayErrorMessage("EXECUTION_CALENDAR value is empty.");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)